cmd/img_mgmt/port/zephyr/src/zephyr_img_mgmt.c

```diff
@@ -36,28 +36,6 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #include <img_mgmt/image.h>
 #include "../../../src/img_mgmt_priv.h"
 
-#if USE_PARTITION_MANAGER
```

Comment:
Where i can find a patch which shows how relation for PM, Trusted_execution and regular build are resolved now?

Reply:
Didn't recive any answer, but hecked that PM build works well, tested with smp_svr sample via BLE.